updating/index.adoc

```diff
@@ -0,0 +1,72 @@
+
+* xref:../updating/installing-update-service.adoc#installing-update-service[Installing and configuring the OpenShift Update Service]:  For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. However, clusters in a restricted network have no way to access public APIs for update information. You can check the following actions:
+
+** xref:../updating/installing-update-service.adoc#update-service-install[Installing the OpenShift Update Service Operator]
+** xref:../updating/installing-update-service.adoc#update-service-delete-service[Uninstalling the OpenShift Update Service Operator]
+** xref:../updating/installing-update-service.adoc#update-service-create-service[Creating an OpenShift Update Service application]
+** xref:../updating/installing-update-service.adoc#update-service-uninstall[Deleting an OpenShift Update Service application]
+
+* xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrading channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are tied to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the openshift-install binary file for a specific version of {product-title} always installs that version. If you do not want the Cluster Version Operator to fetch available updates from the upgrade recommendation service, you can use the `oc adm upgrade channel` command in the `oc` to configure an empty channel. Check out the following information:
+
+** xref:../updating/understanding-upgrade-channels-release.adoc#upgrade-version-paths_understanding-upgrade-channels-releases[Upgrading version paths]
+** xref:../updating/understanding-upgrade-channels-release.adoc#fast-stable-channel-strategies_understanding-upgrade-channels-releases[Understanding Fast and stable channel use and strategies]
+** xref:../updating/understanding-upgrade-channels-release.adoc#restricted-network-clusters_understanding-upgrade-channels-releases[Understanding restricted network clusters]
+** xref:../updating/understanding-upgrade-channels-release.adoc#switching-between-channels_understanding-upgrade-channels-releases[Switching between channels]
+** xref:../updating/understanding-upgrade-channels-release.adoc#conditional-updates-overview_understanding-upgrade-channels-releases[Understanding conditional updates]
+
+* xref:../updating/preparing-eus-eus-upgrade.adoc#preparing-eus-eus-upgrade[Preparing to perform an EUS-to-EUS update]: Due to fundamental Kubernetes design, all {product-title} updates between minor versions must be serialized. You must update from {product-title} 4.8 to 4.9 and then to 4.10. You cannot update from {product-title} 4.8 to 4.10 directly. Check out the following item:
```

Comment:
Add comma: "...4.8 to 4.9, and then to..."

Same comment as before about "Check out..."

Reply:
Addressed